packages/db/postgres/client.js

```diff
@@ -616,8 +617,8 @@ export class PostgresClient {
    * @return {Promise<import('../db-client-types').AuthKey>}
    */
   async getKey (issuer, secret) {
-    /** @type {{ error: Error } */
-    const { data, error } = await this._client
+    /** @type {{ error: PostgrestError } */
+    const { data, error } = (await this._client
```

Comment:
I added the comment in the wrong place, moving it here.

In this particular instance that's actually not required, the problem is our type definition is missing the `data` field, I'll change that.

However I've done a similar change in 2 other instances and the aim was typecasting as described [here]((https://docs.joshuatz.com/cheatsheets/js/jsdoc/#simple-casting).)

But by reviewing the other 2 changes ( [this one](https://github.com/web3-storage/web3.storage/pull/611/files#diff-04117e09fe44d5643c6812b42b94359638abe1e2c2e1e2ba64065c6ca3228819R107) and [this one](https://github.com/web3-storage/web3.storage/pull/611/files#diff-04117e09fe44d5643c6812b42b94359638abe1e2c2e1e2ba64065c6ca3228819R356)) instead of casting we should change the client call and add `.single()` instead.

`.rpc()` returns a `PostgrestFilterBuilder<any>` while we should expect a `PostgrestSingleResponse<any>` instead given we're assuming a single result. Isn't it? (Which also means we wouldn't need to cast)

What you think?

Reply:
I was expecting the `.rpc` call to return a single value instead. But I just confirmed that `createUpload` calls single.

So, this will not change the actual value that is returned, but the types will work. Correct? If so, I think that makes sense to change this and add `.single()`

Reply:
indeed